### PR TITLE
DWR-1060 Initial schema for migrate and reporting olap

### PR DESCRIPTION
**Pull request by agorina**

### CODE REVIEW

File: aggregate_reporting/sql/README.md [hunk 1]
```diff
@@ -1,4 +0,0 @@
-## Redshift aggregate reporting schema
-
-To set up IntelliJ with the redshift database connection:
-https://stackoverflow.com/questions/32319052/connect-intellij-to-amazon-redshift
```
Comment (agorina) [inline aggregate_reporting/sql/README.md]: This file was moved.

File: build.gradle [hunk 2]
```diff
@@ -3,9 +3,11 @@ import java.text.MessageFormat
 buildscript {
     repositories {
         jcenter()
+        maven { url 'http://redshift-maven-repository.s3-website-us-east-1.amazonaws.com/release' }
     }
     dependencies {
         classpath 'mysql:mysql-connector-java:5.1.41'
+        classpath 'com.amazon.redshift:redshift-jdbc42-no-awssdk:1.2.8.1005'
```
Comment (agorina) [inline build.gradle]: I am not included redshift into the rest of this file, but I have it running locally. Will do once I am confident that it does not break anything else.

File: build.gradle [hunk 3]
```diff
@@ -25,21 +27,21 @@ if (project.hasProperty("buildNumber")) {
...
-               info: org.flywaydb.gradle.task.FlywayInfoTask,
-               validate: org.flywaydb.gradle.task.FlywayValidateTask,
-               baseline: org.flywaydb.gradle.task.FlywayBaselineTask,
-               repair: org.flywaydb.gradle.task.FlywayRepairTask]
+def flywayActions = [migrate : org.flywaydb.gradle.task.FlywayMigrateTask,
+                     clean   : org.flywaydb.gradle.task.FlywayCleanTask,
+                     info    : org.flywaydb.gradle.task.FlywayInfoTask,
+                     validate: org.flywaydb.gradle.task.FlywayValidateTask,
+                     baseline: org.flywaydb.gradle.task.FlywayBaselineTask,
+                     repair  : org.flywaydb.gradle.task.FlywayRepairTask]
```
Comment (agorina) [inline build.gradle]: Just ran a formatter.

File: migrate_olap/sql/V1_0_0_0__ddl.sql [hunk 4]
```diff
@@ -0,0 +1,131 @@
+/*
```
Comment (agorina) [inline migrate_olap/sql/V1_0_0_0__ddl.sql]: This is schema for the migrate status and spring batch for the olap migrate.

File: reporting_olap/sql/README.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+## Redshift aggregate reporting schema
+
+#### To set up IntelliJ with the redshift database connection:
+https://stackoverflow.com/questions/32319052/connect-intellij-to-amazon-redshift
+
+####  To configure a search path in Redshift:
+SHOW search_path;   <-- usually returns "$user, public"
+
+ALTER DATABASE <database_name> SET search_path TO schema1,schema2;
+ALTER ROLE <role_name> SET search_path TO schema1,schema2;
```
Comment (agorina) [inline reporting_olap/sql/README.md]: @malaffoon, thank you for figuring this out.

File: migrate_olap/sql/V1_0_0_0__ddl.sql (hunk 4 above)
Comment (jtreuting) [reply migrate_olap/sql/V1_0_0_0__ddl.sql]: does this live in redshift?

**Comment by agorina** [comment]
The folder is named `reporting_olap`. Do not know why git does not show it.

File: migrate_olap/sql/V1_0_0_0__ddl.sql (hunk 4 above)
Comment (agorina) [reply migrate_olap/sql/V1_0_0_0__ddl.sql]: No, we have to find a home for it. 
For now the idea is the warehouse db. Will talk about it later today.

**Pull request merged**